source/get-started/download-and-install.txt

```diff
@@ -4,6 +4,13 @@
 Download and Install
 ====================
 
+.. note:: Alternative Installation Methods
+
+   The following guide shows you how to install {+driver-short+} using
+   `pip <https://pip.pypa.io/en/stable/installation/>`__. To install 
+   {+driver-short+} from source, see `Install from Source <https://pymongo.readthedocs.io/en/stable/installation.html#installing-from-source>`__
+   in the `API Documentation <https://pymongo.readthedocs.io/en/4.10.1/api/>`__.
```

Comment:
S: Since you already link to the specific page in the API docs, I don't think you need this second link
```suggestion
   in the API documentation.
```